### PR TITLE
[WIP] Fixes issue in input feild of creating a new participant team 

### PR DESCRIPTION
**Pull request by sachinmukherjee**
Fixes https://github.com/Cloud-CV/EvalAI/issues/2017

@RishabhJain2018 @deshraj Please review it.

### CODE REVIEW

File: frontend/src/js/controllers/teamsCtrl.js [hunk 1]
```diff
@@ -179,13 +179,13 @@
             parameters.url = 'participants/participant_team';
             parameters.method = 'POST';
             parameters.data = {
-                "team_name": vm.team.name,
-                "team_url": vm.team.url
+                "team_name": vm.new.team.name,
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/teamsCtrl.js]: Can you please change `vm.new.team.name` to `vm.createNewTeam.name` and `vm.createNewTeam.url`

File: frontend/src/js/controllers/teamsCtrl.js [hunk 2]
```diff
@@ -179,13 +179,13 @@
...
+                "team_url": vm.new.team.url
             };
             parameters.token = userKey;
             parameters.callback = {
                 onSuccess: function() {
-                    $rootScope.notify("success", "Team " + vm.team.name + " has been created successfully!");
+                    $rootScope.notify("success", "Team " + vm.new.team.name + " has been created successfully!");
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/teamsCtrl.js]: Same as above!

File: frontend/src/views/web/teams.html [hunk 3]
```diff
@@ -66,13 +66,13 @@ <h6><b>Team Members:</b></h6>
                 <div class="ev-md-container ev-card-body new-team-card">
                     <form ng-submit="teams.createNewTeam()">
                         <div class="input-field align-left">
-                            <input id="name" type="text" class="validate text-dark-black dark-autofill w-400" ng-model="teams.team.name" ng-pattern="/^[\w -.]*$/" focus-if>
+                            <input id="name" type="text" class="validate text-dark-black dark-autofill w-400" ng-model="teams.new.team.name" ng-pattern="/^[\w -.]*$/" focus-if>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/teams.html]: Same as above!

File: frontend/src/views/web/teams.html [hunk 4]
```diff
@@ -66,13 +66,13 @@ <h6><b>Team Members:</b></h6>
...
                             <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
                             <label for="name" data-error="wrong" data-success="right">Team Name*</label>
                             <div class="wrn-msg text-highlight" ng-show="teams.team.error">{{teams.team.error}}</div>
                         </div>
                         <div class="input-field align-left">
-                            <input id="url" type="text" class="validate text-dark-black dark-autofill w-400" ng-model="teams.team.url" focus-if>
+                            <input id="url" type="text" class="validate text-dark-black dark-autofill w-400" ng-model="teams.new.team.url" focus-if>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/teams.html]: Same as above!

**Comment by sachinmukherjee** [comment]
Will do the requested changes

File: frontend/src/js/controllers/teamsCtrl.js (hunk 1 above)
Comment (sachinmukherjee) [reply frontend/src/js/controllers/teamsCtrl.js]: While changing `vm.new.team.name` to `vm.createNewTeam.name` it is throwing error
`TypeError: Cannot assign to read only property 'name' of function 'function() {
            vm.isExistLoader = true;
            vm.loaderTitle = '';
            vm.loaderContain...<omitted>... }'`
@RishabhJain2018 Should I need to change something else

Comment (sachinmukherjee) [reply frontend/src/js/controllers/teamsCtrl.js]: I think createNewTeam is the function name in the controller so we cannot use that name again as a variable name. @RishabhJain2018

**Comment by sachinmukherjee** [comment]
@RishabhJain2018 Please review it. You have asked me to change `new.team` to `createNewTeam` but as the function name inside of out controller was same it was conflicting and was creating issues. so I have changed the variable name from `new.team` to `createTeam`

**Pull request closed without merging**